src/components/backend-ai-session-list.ts

```diff
@@ -1330,8 +1415,37 @@ export default class BackendAiSessionList extends BackendAIPage {
   sessionInfoRenderer(root, column?, rowData?) {
     render(
       html`
+        <style>
+          #session-name-field {
+            display: block;
+            margin-left: 16px;
+          }
+          #session-rename-field {
+            display: none;
+            --mdc-text-field-fill-color: transparent;
+            --mdc-text-field-disabled-fill-color: transparent;
+            --mdc-ripple-color: transparent;
+            width: min-content;
+          }
+          #session-rename-icon {
+            --mdc-icon-size: 20px;
+          }
+        </style>
         <div class="layout vertical start">
-          <div>${rowData.item[this.sessionNameField]}</div>
+          <div class="horizontal center center-justified layout">
+            <div id="session-name-field">${rowData.item[this.sessionNameField]}</div>
```

Comment:
Everything seems fine, but one thing bothers me.
If the session name is almost reached the limit (64 characters), it's bit cumbersome since the user always needs to resize the sessioninfo column first.

How about using `pre` tag instead of `div` tag with style definition like this:
```
#session-name-field {
            display: block;
            margin-left: 16px;
            white-space: pre-wrap;
            word-break: break-all;
          }
...
<pre id="session-name-field">${rowData.item[this.sessionNameField]}</pre>
```

**screenshot**:
![wrapped-session-name mov](https://user-images.githubusercontent.com/46954439/148804692-de9157ea-a4e2-4bb0-a626-0b87cdc52d43.gif)

Reply:
you're right it'll be better! I'll apply that right away.